aslam_cv_matcher/src/gyro-two-frame-matcher.cc

```diff
@@ -21,7 +28,9 @@ GyroTwoFrameMatcher::GyroTwoFrameMatcher(
     kImageHeight(image_height),
     matches_kp1_k_(matches_with_score_kp1_k),
     is_keypoint_kp1_matched_(kNumPointsKp1, false),
-    iteration_processed_keypoints_kp1_(kNumPointsKp1, false) {
+    iteration_processed_keypoints_kp1_(kNumPointsKp1, false),
+    small_search_distance_px_(FLAGS_gyro_matcher_small_search_distance_px), 
+    large_search_distance_px_(FLAGS_gyro_matcher_large_search_distance_px) {
```

Comment:
add check if both these variables are greater than 0 and if `large_search_distance_px_ > small_search_distance_px_`